.github/workflows/cd.yaml

```
@@ -15,30 +15,37 @@ concurrency:
 
 jobs:
 
-  cd_appimage_x86_64:
+  cd_appimage_x64:
     name: Linux
     runs-on: ubuntu-20.04
     env:
-      ARCH: x86_64
-      OUTPUT: endless-sky-x86_64-continuous.AppImage
+      OUTPUT: Endless_Sky-continuous-x86_64.zip
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      CXX: sccache g++
     steps:
       - uses: actions/checkout@v3
       - name: Install dependencies
         run: |
           sudo rm /etc/apt/sources.list.d/* && sudo dpkg --clear-avail # Speed up installation and get rid of unwanted lists
-          sudo apt-get update
-          sudo apt-get install -y --no-install-recommends libsdl2-dev libpng-dev libjpeg-turbo8-dev libopenal-dev libmad0-dev libglew-dev libgl1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev uuid-dev
-          python3 -m pip install --user scons
+          sudo apt update
+          sudo apt install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev
```

```suggestion
          sudo apt-get update
          sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev
```
Should still be `apt-get` for automated invocations